src/css/_private-variables.scss

```diff
@@ -1,15 +1,15 @@
-$primary-text: #fff !default;
-// currently just used for visited links in errors and such.
-$secondary-text: #F4A460 !default;
+// Text, icons, hover states
+$primary-foreground-color: #fff;
 
-$primary-bg: #000 !default;
-$secondary-bg: lighten($primary-bg, 35%) !default;
+// Control backgrounds (control bar, big play, menus)
+$primary-background-color: #2B333F !default;
+$primary-background-transparency: 0.7 !default;
 
-$text-font-family: Arial, Helvetica, sans-serif !default;
-$base-font-size: 10px !default;
+// Hover states, slider backgrounds
+$lighter-background-color: lighten($primary-background-color, 33%) !default;
+$lighter-background-transparency: 0.5 !default;
 
-$control-bar--bg: #2B333F !default;
-$control-bar--transparency: 0.5 !default;
+$text-font-family: Arial, Helvetica, sans-serif !default;
 
 $big-play-button--width: 3em !default;
```

Comment:
Just to clear this up after our conversation, add a comment of some kind here solidifying the naming structure for component-specific variables (`$component-name--attribute`).